test/parallel/test-http-upgrade-client.js

```diff
@@ -60,7 +61,8 @@ srv.listen(0, '127.0.0.1', common.mustCall(function() {
       ['Origin', 'http://www.websocket.org']
     ]
   ];
-  let left = headers.length;
+  const countdown = new Countdown(10, () => srv.close());
```

Comment:
Why 10 ? It should be `headers.length`.

Reply:
@daxlab : My bad.. thanks much for the correction. I've updated the PR. Kindly review!